wdl/imputation.wdl

```diff
@@ -558,12 +558,21 @@ task duplicates {
             awk -v batch=$batch '{OFS="\t"; print batch,$0}' $batch.samples_include.txt >> all_batches.samples_include.txt
             awk -v batch=$batch '{OFS="\t"; print batch,$0}' $batch.samples_exclude.txt >> all_batches.samples_exclude.txt
         done
+          #Rplot fix for batches that have empty exclusion list
+          awk -v batch=$batch 'BEGIN {FS="\t";OFS="\t"} { if ($1==batch) print "exists"; else print "not_exist"}' all_batches.samples_exclude.txt > file1.txt
```

Comment:
$batch will here be the last batch that was iterated in the previous for loop, probably not intentional?